Dry run updates (part 2) (#345)
aws-samples/aws-hpc-tutorials: content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md

echo ${CUSTOM_AMI}
```

> #### 3. Retrieve NCAR WRF v4 AMI.

Reviewer: Why this part is removed?

Author: We already have the AMI ID from the script, this just shows how we got it. We also are not grabbing it from the NCAR account. So if we put this back in then I'll remove "NCAR". Thoughts?

Reviewer: yes. we need to explain from where is it coming from to avoid attendees to believe this is coming with the default parallelcluster AMI.

echo ${CUSTOM_AMI}
```

#### 3. Retrieve WRF v4 AMI.

> We have built an Amazon Machine Image (AMI) that contains a compiled version of WRF v4.

Reviewer: ```suggestion
An Amazon Machine Image (AMI) that contains a compiled version of WRF v4 has been built for this workshop.
```

  SlurmQueues:
    - Name: queue0
      ComputeResources:
        - Name: c524xl
          MinCount: 0
          MaxCount: 2
          InstanceType: c5.24xlarge
          DisableSimultaneousMultithreading: true
          Efa:
            Enabled: false
        - Name: c518xl
          MinCount: 0
          MaxCount: 2

>           InstanceType: c5.18xlarge

Reviewer: we said 24xl

Author: The quip had both 24 and 18. This change puts 24 first (swapping the order). Do we not want 18 as a fallback?

Reviewer: ok I misread this. Sorry.
